src/components/circle.vue

```diff
@@ -45,16 +71,92 @@ const events = [
     'mouseout',
     'mouseover',
     'mouseup',
+    'radius_changed',
     'rightclick'
 ]
-
-export default {
-    mixins: [MapElementMixin, getPropsValuesMixin],
+const getLocalField = function (self, field){
+  return (typeof self.$options.propsData[field] !== 'undefined')?self[field]:self.circleObj[field];
+};
+const setLocalField = function (self, field, value){
+  self.circleObj[field] = value;
+  self.$emit(field+'_changed', value);
+  self.$nextTick(function (){
+    self.circleObj[field] = getLocalField(self, field);
+  });
+};
+export default MapComponent.extend({
+    mixins: [getPropsValuesMixin],
     props: props,
-    version: 2,
-
-    render() { return '' },
-
+    data(){
+      return {
+        circleObj:{
+          center:{},
+          radius:null,
+          bounds:null,
+          draggable:null,
+          editable:null,
+          options:{}
+        }
+      };
+    },
+    computed:{
+        local_center:{
```

Comment:
Could you explain what kind of functionality you are trying to achieve with local_center that you cannot achieve with just props and event?